main.js

```diff
@@ -0,0 +1,51 @@
+// DOM utility functions
+const addClass = (el, newClass) => {
+  el.className += ` ${newClass}`;
+};
+
+const removeClass = (el, oldClass) => {
+  const currentClasses = el.className.split(" ");
+
+  el.className = currentClasses.filter(names =>
+    names.trim() !== "" && names !== oldClass
+  ).join(" ");
+};
+
+// Search Script
+(() => {
```

Comment:
When not using jQuery's `$(document).ready`, the best way:
http://stackoverflow.com/questions/9899372/pure-javascript-equivalent-to-jquerys-ready-how-to-call-a-function-when-the

Moved the script tag to the bottom of the body tag so that the search field will be loaded by the time this executes.